src/Host.cpp

```diff
@@ -3612,6 +3612,25 @@ bool Host::setBackgroundColor(const QString& name, int r, int g, int b, int alph
     return false;
 }
 
+QColor Host::getBackgroundColor(const QString& name) const
+{
+    if (!mpConsole) {
```

Comment:
What about something for the *"main"* `TMainConsole` instance?

:thinking: 

Wouldn't it be a better code structure to have the `TLuaInterpreter` function call code in the `(TMainConsole*) Host::mpConsole` instance that looks up the passed `name` in its own `QMap`s - and it can return it's own value for `mBgColor` if called with an empty or `main` value. Of course it can also generate the `nil` + error message needed if the name does not match anything - which the currently propose Lua function does not do!

Reply:
Same as above, leaving it for consistency. Wouldn't mind this structure though.